src/python/WMQuality/Emulators/SiteDBClient/MockSiteDBApi.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+
+"""
+Version of SiteDB.SiteDBJSON intended to be used with mock or unittest.mock
+"""
+from __future__ import (division, print_function)
+
+from WMCore.WMBase import getTestBase
+from RestClient.ErrorHandling.RestClientExceptions import HTTPError
+import os
+import json
+
+# Read in the data just once so that we don't have to do it for every test (in __init__)
+mockData = {}
+globalFile = os.path.join(getTestBase(), '..', 'data', 'Mock', 'SiteDBMockData.json')
+print(globalFile)
+
+try:
+    with open(globalFile, 'r') as mockFile:
+        mockData = json.load(mockFile)
+except IOError:
+    mockData = {}
+
+
+def mockGetJSON(dummySelf, callname, filename='result.json', clearCache=False, verb='GET', data=None):
+    """
+    _getJSON_
+
+    retrieve JSON formatted information given the service name and the
+    argument dictionaries
+
+    TODO: Probably want to move this up into Service
+    """
+    #print("Using mock SiteDB getJSON")
+    #Don't set default parameter to empty dict
+    if data is None:
+        data = {}
+
+    #Build args
+    kwargs = {}
+    args = ['callname', 'filename', 'clearCache', 'verb', 'data']
+    for arg in args:
+        kwargs.setdefault(arg, eval(arg))
```

Comment:
Find a way to do this without eval(). Eval is dangerous and we don't want it anywhere we can help it.